docs/configs/index.md.jsx

```diff
@@ -64,6 +64,7 @@ const docsSource = {
   '2.0.5': docs205Config,
   '3.0.0': docs300Config,
   '3.0.1': docs301Config,
+  '3.0.2': docs302Config,
```

Comment:
So as this file, we should change branch `dev` before we announce instead of this step

Reply:
you can see more detail in section *Update Document*

Reply:
> you can see more detail in section _Update Document_

Looks like we need some updates for the docs. I will submit a PR later to add more instructions. 
![image](https://user-images.githubusercontent.com/34905992/201569718-f36e8365-5c36-4338-baf8-4c9a011c2ba6.png)